R/prevalence.R

```diff
@@ -0,0 +1,94 @@
+#' @title Taxa Prevalence 
+#' @description Simple prevalence measure.
+#' @param x A vector, data matrix or \code{\link{MicrobiomeExperiment}} object
+#' @param sort Sort the groups by prevalence
+#' @param count Logical. Indicate prevalence as fraction of samples
```

Comment:
Name changes suggestion: `as_relative`

Reply:
Can you clarify. Do you refer to an argument name?

Reply:
Yes. If I see a variable named `count`, I am thinking numeric, not logical. `as_relative` is immediately recognizable as `logical`

Reply:
Ah yes I see. True. 

Should we use the term `compositional` or `relative`? This is also a generic decision point.

Reply:
I like `as_relative`

Reply:
Actually here relative can refer to either compositional abundances in prevalence calculation; or to prevalence measure itself (relative frequency or absolute count). For clarity, this function now always returns prevalence as a relative frequency. The prevalences can be calculated from absolute or relative data (as_relative option).

Reply:
Another point: this function had `include.lowest` and `as_relative` as arguments. I have now changed to `include_lowest` for consistency. This should be unified across the package/s.